tests/Directory.Build.props

```diff
@@ -20,10 +20,6 @@
     <OutputType Condition="$(MSBuildProjectName.EndsWith('.Tests'))">Exe</OutputType>
   </PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="GitHubActionsTestLogger" PrivateAssets="all" IncludeAssets="runtime; build; native; contentfiles; analyzers" />
```

Comment:
are we losing anything from dropping this?

Reply:
Yes.

From GitHubActionsTestLogger README, it does things similar to the following:

![image](https://github.com/user-attachments/assets/816a04f3-a414-4a90-9b69-9f596b2fa3b7)

![image](https://github.com/user-attachments/assets/ceb2ed6d-efd3-4517-8909-d8eb71999a5f)

We can discuss internally if it's important for you though.

Reply:
It's OK and we can worry about this later/discuss internally later.

Reply:
I'm glad we're removing this package.